src/routers/auth.js

```diff
@@ -0,0 +1,19 @@
+const express = require('express');
+const router = express.Router();
+const CookieName = "blogCookie";
+
+router.use(function timeLog(req, res, next) {
+    next()
```

Comment:
?) why?)

Reply:
It's a relic. I've got to update it, thanks )